README.rst

```diff
@@ -0,0 +1,24 @@
+=====
+Django data access log
+=====
+
+Django data access log is a simple Django app to log the access into a JRF viewset.
```

Comment:
Django data access log is a simple Django app to log the data accessed in a DRF viewset.